plugins/auth-backend/src/lib/catalog/CatalogIdentityClient.ts

```diff
@@ -45,7 +45,7 @@ export class CatalogIdentityClient {
 
   constructor(options: {
     catalogApi: CatalogApi;
-    tokenManager: TokenManager;
```

Comment:
Just keeping things nonbreaking I presume

Reply:
Yep, in the new system we'll use the actual service, in the old we still end up at https://github.com/backstage/backstage/blob/b9dca423cdf8c50ecade8ec7392220fb263c7ea5/packages/backend-common/src/compat/auth/createLegacyAuthAdapters.ts#L105